src/test/objectstore/test_bluefs.cc

```diff
@@ -49,7 +49,11 @@ TEST(BlueFS, mkfs) {
   uint64_t size = 1048576 * 128;
   string fn = get_temp_bdev(size);
   uuid_d fsid;
-  BlueFS fs(g_ceph_context);
+  g_ceph_context->_conf.set_val(
```

Comment:
nit: passing 1MB to ctor and replacing references to conf->bluefs_alloc_size with fs.get_bluefs_alloc_size()  wherever they are used seems a more elegant approach to me...
Can be done later though....